src/main/webapp/app/exercises/quiz/manage/statistics/quiz-statistic/quiz-statistic.component.ts

```diff
@@ -115,13 +115,17 @@ export interface DataSetProvider {
      */
     loadDataInDiagram(): void;
 }
-
+// wrapper function to ensure that other ng2-implementations still work; should be removed as soon as not used anymore
```

Comment:
I would prefer this to be a real documentation comment, note that the argument needs to be covered aswell, so this suggestion should not be committed directly but should just give an idea of what I mean.
```suggestion
/**
* wrapper function to ensure that other ng2-implementations still work; should be removed as soon as not used anymore
*/
```

Reply:
I changed the comment in d0f2078